wandb/integration/openai/__init__.py

```diff
@@ -1,3 +1,12 @@
-__all__ = ("autolog",)
+__all__ = ("autolog", "WandbLogger")
+
+from wandb import util
+
+_openai = util.get_module("openai")
```

Comment:
i assume `openai` module is required. so you should look at the `get_module` and add a message in case it is not imported (see the required field), also please make sure that lazy import is what you want!